src/pages/home/report/ReportActionCompose.js

```diff
@@ -644,7 +644,8 @@ class ReportActionCompose extends React.Component {
                                             displayFileInModal(file);
                                             this.setState({isDraggingOver: false});
                                         }}
-                                        style={[styles.textInputCompose, this.props.isComposerFullSize ? styles.textInputFullCompose : styles.flex4]}
+                                        style={[styles.textInputCompose, this.props.isComposerFullSize ? styles.textInputFullCompose : styles.inputComposerPadding]}
+                                        defaultValue={this.props.comment}
```

Comment:
Why this?

Reply:
Apart from `flex4`, I had to add more styling and so `styles.inputComposerPadding`

Reply:
I'm asking about why defaultValue prop was added here?